src/ValueObject/Route/V2.php

```diff
@@ -27,6 +27,6 @@ public function __construct()
     {
         $this->paramNameList = array();
         $this->identifier    = '/v2';
-        $this->pattern       = '/^\/v2\/*$/';
+        $this->pattern       = '/\/v2\/*$/';
```

Comment:
This change might cause misdetection of uris. E.g. "/products/12/v2/" would be detected as "/v2/" as well.